mnemocards/maketsv.py

```diff
@@ -111,36 +111,40 @@ def format_explanations(list_obj, explanation_name, main_translation=None):
 
 
 def prepare_card_fields(translation):
-    main_translation = translation.extra_data["translation"]
-    full_trans = translation.extra_data["all-translations"]
-    definitions_trans = translation.extra_data["definitions"]
+    try:
+        main_translation = translation.extra_data["translation"]
+        full_trans = translation.extra_data["all-translations"]
+        definitions_trans = translation.extra_data["definitions"]
 
-    ylw = main_translation[0][0]
-    lylw = translation.origin
+        ylw = main_translation[0][0]
+        lylw = translation.origin
 
-    if ylw.lower() == lylw.lower() and full_trans is None:
-        return None
+        if ylw.lower() == lylw.lower() and full_trans is None:
```

Comment:
Here it checks if the translation equals original words and if there are no synonyms produced. So in cases with words like 'css' or some other it will return `None` and thus it doesn't need `try\except` you proposed